js/components/deepMerge.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * WIKIA CHANGE
```

Comment:
if the change is to add this whole file please write "Added by Wikia" otherwise mark the end of change